src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -137,20 +136,28 @@ public RequestTranscriptionResponse saveTranscriptionRequest(
         UserAccountEntity userAccount = getUserAccount();
         TranscriptionStatusEntity transcriptionStatus = getTranscriptionStatusById(REQUESTED.getId());
 
-        TranscriptionEntity transcription;
-        try {
-            transcription = saveTranscription(
-                userAccount,
-                transcriptionRequestDetails,
-                transcriptionStatus,
+        if (transcriptionRequestDetails.getStartDateTime() != null && transcriptionRequestDetails.getEndDateTime() != null) {
+            List<TranscriptionEntity> matchingTranscriptions = transcriptionRepository.findByHearingIdTypeStartAndEnd(
+                transcriptionRequestDetails.getHearingId(),
                 getTranscriptionTypeById(transcriptionRequestDetails.getTranscriptionTypeId()),
-                getTranscriptionUrgencyById(transcriptionRequestDetails.getUrgencyId()),
-                isManual
+                transcriptionRequestDetails.getStartDateTime(),
+                transcriptionRequestDetails.getEndDateTime()
             );
-        } catch (DataIntegrityViolationException e) {
-            throw new DartsApiException(TranscriptionApiError.DUPLICATE_TRANSCRIPTION);
+            if (!matchingTranscriptions.isEmpty()) {
+                String errorDetail = "{ \"transcription_id\": " + matchingTranscriptions.get(0).getId().toString() + " }";
```

Comment:
Does the transcriptionId need to be represented in the response as JSON?

If so, we could enhance the DartsApiException to allow us to add arbitrary fields to the error response. So here it could be something like:

```json
{
  "type": "TRANSCRIPTION_107",
  "title": "A transcription already exists with these properties",
  "status": 409,
  "duplicate_transcription_id": "2"
}
```

Reply:
I'll draft up a quick patch and will send it your way.